web/locales/de.json

```diff
@@ -1097,10 +1099,12 @@
             "zoneName": "Russland"
         },
         "RU-EU": {
-            "zoneName": "Russland"
+            "zoneName": "Russland",
+            "countryName": "Russland"
         },
         "RU-AS": {
-            "zoneName": "Russland"
+            "zoneName": "Russland",
+            "countryName": "Russland"
         },
```

Comment:
There's been a few updates to RU zones, would you mind updating the translations to reflect this? Here's the names from `en.json` translation :)

```
		  "RU-EU": {
            "countryName": "Russia",
            "zoneName": "Arctic"
          },
          "RU-AS": {
            "countryName": "Russia",
            "zoneName": "East"
          },
          "RU-FE": {
              "countryName": "Russia",
              "zoneName": "Far East"
          },
          ```

Reply:
Yes, I can do that